payments/tests/__init__.py

```diff
@@ -0,0 +1,21 @@
+import stripe
+
+from ..utils import transaction_amounts
+
+
+def sandbox_charge(offer_amount):
```

Comment:
Note, move to `payments/management/commands/spot_test_stripe_charge.py` `Command`